packages/botonic-react/src/components/message.jsx

```diff
@@ -95,9 +95,9 @@ export const Message = props => {
     style,
     imageStyle,
     timestamps = true,
-    markdown = true,
     ...otherProps
   } = props
+  const markdown = props.markdown
```

Comment:
pardon my ignore of botonic-react, I don't understand why this change affects only texts

Reply:
Maybe the description now is clearer. The fact here is that from within the `Text` component, we create the message as this. Inside textProps we have {markdown:true} set by default:
```
<Message json={serialize(textProps)} {...textProps} type={INPUT.TEXT}>
        {textProps.children}
</Message>
```
When destructuring the message in Message component, we don't want to have this prop for all messages, but only for the Components that have this prop (in our case Text)